src/classes/Standardizer/plugins/Google/getters/getNotes.ts

```diff
@@ -11,6 +11,12 @@ interface GoogleNotes {
   userEditedTimeStampUsec: number
 }
 
+function computeCreationDate(file:string, note:GoogleNotes) {
+  return new Date(
+    file.split('Takeout/Keep/')[1].replaceAll('_', ':').split('.json')[0],
+  ) ?? new Date(note.userEditedTimeStampUsec / 1000)
```

Comment:
`new Date(note.userEditedTimeStampUsec / 1000)` is not really the creation date. As you made `creationDate` property optional in `Note` schema, you can use `undefined` as fallback

Reply:
I could. I'd rather have a date even if it's not the creation date, instead of having nothing. 
Our goal is to display as much data as possible. I think it's better falling back to the last edited date.